multi_block_core.dic

```diff
@@ -56,6 +56,156 @@ save_MULTIBLOCK_CORE
+         _pd_phase.id "Second phase"
+         #cell parameters and atom positions
+         #...
+         
+         data_diffraction_data1
+         _pd_diffractogram.id "First dataset"
+         #diffracted intensities vs angle
+         #...
+         
+         data_information
+         #wavelength, specimen preparation, temperature, pressure...
+         #...
+;
+;
+         Four data blocks for a powder diffraction experiment.
+         The absence of an _audit_dataset.id value does not preclude that all 
+         or some of the four data blocks may to be interpretted together. The 
+         consumer of the information may be able to infer that they belong
+         together due to the presence of other information.
+;
+;
+         data_structure1
+         _audit_dataset.id  34e58dee-900c-47cf-8f8c-1f52e5a9a3b9
+         _pd_phase.name "First phase"
+         #cell parameters and atom positions
+         #...
+         
+         data_calibration_structure
+         loop_ 
+           _audit_dataset.id  
+           34e58dee-900c-47cf-8f8c-1f52e5a9a3b9
+           2d7ee621-d916-4302-9045-ec68f56add45
+           255bb051-a3d5-43f0-8384-fa8ccb2ce3c7
+         _pd_phase.id "Calibration phase"
+         #cell parameters and atom positions
+         #...
+         
+         data_diffraction_data
+         _audit_dataset.id  34e58dee-900c-47cf-8f8c-1f52e5a9a3b9
+         _pd_diffractogram.id "First dataset"
+         #diffracted intensities vs angle
+         #...
+;
+;
+         Three data blocks for a powder diffraction experiment.
+         By the presence of an identical _audit_dataset.id value, it is made
+         clear that all data blocks are to be interpretted together as 
+         a whole. The presence of multiple _audit_dataset.id values for the
+         second data block enables it to be included in more than one dataset.
+
+         Additionally, it is clear from the presence of different 
+         _audit_dataset.id values, that the diffractogram _pd_diffractogram.id 
+         "First dataset" in the first example is different to the diffractogram
+         _pd_diffractogram.id "First dataset" in this example.
+;
+
+save_
+
+save_audit_dataset.id
+
+    _definition.id                '_audit_dataset.id'
+    _definition.update            2025-04-14
+    _description.text
+;
+    Globally unique identifier for data blocks that are meant to be interpreted 
+    together as a whole.
+
+    Data blocks which belong together, for instance, as they form part of the 
+    same experiment, are given an identical _audit_dataset.id value.
+
+    Data blocks, such as calibration information, can be designated with more
+    than one _audit_dataset.id value.
+
+    Data blocks which do not contain the same unique _audit_dataset.id may still 
+    be related, depending on the presence of other data names and values. Data
+    blocks which contain different _audit_dataset.id values are not related in 
+    a CIF-sense.
```

Comment:
This paragraph is confusing. The first sentence says that data blocks with different `_audit_dataset.id` may still be related, then the next one says the opposite. I think the intended meaning is that data blocks for which `_audit_dataset.id` is absent may still be related to one another?

Reply:
Yes, that first sentence in the last paragraph is wrong.


It should read:
> Data blocks which do not contain a _audit_dataset.id may still be related.